Ex4_2.html

```diff
@@ -0,0 +1,130 @@
+<!DOCTYPE>
+<html>
+
+<head>
+<title>Defining Object</title>
+<script type="text/javascript">
+
+  function ifExist(){
+      var someObjectID = prompt("Please, enter object ID or ID.element you interesting in.", "");
+      var inputArray = someObjectID.split(".");
+      var someObject = document.getElementById(inputArray[0]);
+      if (someObject===null){
+          return console.log("Type of input parameter is " + someObject + " or parameter does not exist.");
+      }
+      /* else if (someObject === undefined){
+          return console.log("Type of input parameter is " + someObject + " or parameter does not exist.");
+          //someObject = prompt(("Type of input parameter is " + objectType + "." +  "\nPlease, enter below object you interesting in:"), ""); 
+          //return ifExist(someObject);
+          } */
+      else {
+          if(inputArray.length==1){
+            var objectType = (typeof(someObject));
+            return console.log("Type of input parameter is " + objectType +".");
+          }
+          else{
+            var someElement = someObject[inputArray[1]-1]
+            if(someElement===null){
+              return console.log("Type of input element is " + someElement + " or element does not exist.");
+            }
+            else{
+              var elementType = (typeof(someElement));
+              return console.log("Type of input element is " + elementType +".");
+            }
+            
+          }
+    
+      }
+    }
+    function selectedObjects(){
+      var checkedElements = []; 
+      var checkBoxs = document.getElementsByClassName("checked");
+      for(var i = 0; i<checkBoxs.length; i++){
+        if(((checkBoxs[i].type == "checkbox")||(checkBoxs[i].type == "radio")) && (checkBoxs[i].checked)){
+          console.log(checkBoxs[i])
+        }
+        else if((checkBoxs[i].type == "text"||checkBoxs[i].type == "password") && (checkBoxs[i].value !="")){
+          console.log(checkBoxs[i])
+        } 
+        else if((checkBoxs[i].selected == true)){
```

Comment:
why we need == true?

Reply:
we define objects in block with "Numbers" was selected by user